graders/c/cgrader/cgrader.py

```diff
@@ -102,7 +102,7 @@ def run_command(
         input: Any | None = None,  # noqa: A002
         sandboxed: bool = True,  # noqa: FBT001
         timeout: float | None = None,
-        env: subprocess._ENV | None = None,
+        env: Any | None = None,
```

Comment:
Can get rid of the union and just do any

Reply:
Or use dict to make it consistent with:
https://github.com/PrairieLearn/PrairieLearn/blob/45a0aedd332ad7891839d316b029d82b8cebe74c/graders/c/cgrader/cgrader.py#L634

Reply:
^ even better!

Reply:
Done in 584c785.